src/sql/workbench/contrib/editData/browser/editDataGridPanel.ts

```diff
@@ -232,10 +232,12 @@ export class EditDataGridPanel extends GridParentComponent {
 				// If a cell is not edited and retrieved direct from the SQL server, it would be in the form of a DBCellValue.
 				// We use the DBCellValue's cleaned displayValue as the text value.
 				returnVal = this.replaceLinebreaks(value.displayValue);
+				returnVal = returnVal.trim();
```

Comment:
What if the user actually wants to add spaces to the beginning/end of the string? Isn't this going to prevent that from happening?